skimage/segmentation/_felzenszwalb_cy.pyx

```diff
@@ -34,13 +35,20 @@ def _felzenszwalb_cython(image, double scale=1, sigma=0.8,
         Larger sigma gives smother segment boundaries.
     min_size : int, optional (default 20)
         Minimum component size. Enforced using postprocessing.
+    multichannel : bool, optional (default: True)
+        Whether the last axis of the image is to be interpreted as multiple
+        channels. A value of False, for a 3D image, is not currently supported.
 
     Returns
     -------
     segment_mask : (N, M) ndarray
         Integer mask indicating segment labels.
     """
-    if image.ndim != 3 or image.shape[2] not in [1, 3]:
```

Comment:
This check isn't necessary because _felzenszwalb.py will always call `np.atleast_3d`.